### ISSUE TITLE
server side access to layers controls

### ISSUE THREAD

**Issue by ComeMaes** [rstudio/leaflet#404]
Hi , 
Is there any way to access the `LayersControls` state from the server code when using leaflet together with Shiny?
Typically it would be great to access the state of `baseGroups`and `overlayGroups` as set by `addLayersControls()`, and retrieve which groups are active, and so on.
I couldn't find a way to do this so far. Could these be provided in the `input$MAPID_******` ? I know the documentation clearly states tiles and controls don't raise events but I can't see why, and in my opinion this would be pretty convenient :)
Thanks!

**Comment by bhaskarvk** [comment]
We could always capture leaflet events for these and emit Shiny events accordingly.

**Comment by ComeMaes** [comment]
Yes. There's this leaflet plugin that does something close: https://github.com/vogdb/Leaflet.ActiveLayers
I don't know if there is a leaflet native or simpler way to do it.

**Comment by jcheng5** [comment]
I could've sworn we already do this, but you're right, we don't.

**Comment by ComeMaes** [comment]
@jrisi256 I have not found any, personally, but haven't tried with latest versions either - although latest documentation does not show any change regarding that issue.

@jcheng5 Any update on this? :)

**Comment by kent37** [comment]
This is implemented but not documented, see #640.
